### PR TITLE
Use the footer on tablets (< lg, > md)

### PR DESCRIPTION
**Pull request by alalonde**

### CODE REVIEW

File: packages/web/components/MobileFooter.tsx [hunk 1]
```diff
@@ -53,7 +53,8 @@ const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
       href={href}
       isExternal={isExternal}
       margin="0 !important"
-      width="7rem"
+      width="7remimport { useBreakpointValue } from '@chakra-ui/react';
```
Comment (hammadj) [inline packages/web/components/MobileFooter.tsx:56]: Looks like this isnt meant to be here

File: packages/web/components/MobileFooter.tsx [hunk 2]
```diff
@@ -166,9 +167,9 @@ export const MobileFooter: React.FC = () => {
         <Stack
           position="fixed"
           left="0"
-          top="0"
+          bottom="5rem"
           width="100vw"
-          height="calc(100vh - 5rem)"
+          height="min(100vh, 36rem)"
```
Comment (hammadj) [inline packages/web/components/MobileFooter.tsx:172]: What's the intention behind these changes?

Comment (alalonde) [reply packages/web/components/MobileFooter.tsx:172]: This prevents the drawer from expanding full-screen on e.g. tablets where it's way too big. When the view height is < 36rem.

**Review by dysbulic — CHANGES_REQUESTED** [review]

**Comment by alalonde** [comment]
@dysbulic Does your PR supercede these changes? I can close this; otherwise let's move it through. @hammadj

**Comment by dysbulic** [comment]
> @dysbulic, does your PR supercede these changes?

Yes. #294 uses the footer version of the drawer all the way up to `lg`.

**Pull request closed without merging**